# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: e6523049b675
# Full Hash: e6523049b6751146641d019b9adc0825473c46aa
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 09:10:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 1 - Move COLR font support functions from gfxUtils into their own class. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152036
# ==============================================================================

diff -r 90f3275f3489 -r e6523049b675 gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Tue Aug 16 12:40:10 2022 +0000
+++ b/gfx/thebes/moz.build	Tue Aug 16 12:48:31 2022 +0000
@@ -17,6 +17,7 @@
 XPIDL_MODULE = "gfx"
 
 EXPORTS += [
+    "COLRFonts.h",
     "DrawMode.h",
     "gfx2DGlue.h",
     "gfxAlphaRecovery.h",
@@ -195,6 +196,7 @@
 
 UNIFIED_SOURCES += [
     "CJKCompatSVS.cpp",
+    "COLRFonts.cpp",
     "gfxAlphaRecovery.cpp",
     "gfxBaseSharedMemorySurface.cpp",
     "gfxBlur.cpp",
